contrib/terraform-testing-infrastructure/README.md

```diff
@@ -0,0 +1,227 @@
+| us_east_1b_subnet | |  true |
+| us_east_1e_subnet | |  true |
+| route53_zone | |  true |
+| ami_owner | |  true |
+| ami_name_pattern | |  true |
+| authorized_ssh_keys | | true |
+| zookeeper_dir | /data/zookeeper | true |
+| hadoop_dir | /data/hadoop | true |
+| accumulo_dir | /data/accumulo | true |
+| maven_version | 3.8.4 |  true |
+| zookeeper_version | 3.5.9 | true |
+| hadoop_version | 3.3.1 | true |
+| accumulo_version | 2.1.0-SNAPSHOT | true |
+| accumulo_repo | https://github.com/apache/accumulo.git | true |
+| accumulo_branch_name | main | true |
+| accumulo_testing_repo | https://github.com/apache/accumulo-testing.git | true |
+| accumulo_testing_branch_name | main | true |
+| local_sources_dir | | false |
+
+### Configuration
+
+You will need to provide values for the required variables with no default value. You can also override the default values by supplying new values. There are several [ways](https://www.terraform.io/language/values/variables#assigning-values-to-root-module-variables) to do this. Below is an example of a JSON file that contains values, which can be placed in the `aws` directory with a name that ends with `.auto.tfvars.json`, that will be applied during the `terraform apply` command. 
+
+```
+{
+  "security_group": "sg-ABCDEF001",
+  "route53_zone": "some.domain.com",
+  "us_east_1b_subnet": "subnet-ABCDEF123",
+  "us_east_1e_subnet": "subnet-ABCDEF124",
+  "ami_owner": "000000000001",
+  "ami_name_pattern": "MY_AMI_*",
+  "authorized_ssh_keys": [
+    "ssh-rsa dev_key_1",
+    "ssh-rsa dev_key_2"
+  ]
+}
+```
+
+## AWS Resources
+
+This Terraform configuration creates:
+
+  1. An EFS filesystem with IP addresses in the `${us_east_1b_subnet}` and `${us_east_1e_subnet}` subnets
+  2. `${instance_count}` EC2 nodes of `${instance_type}` with the latest AMI matching `${ami_name_pattern}` from the `${ami_owner}`. Each EC2 node will have a `${root_volume_gb}`GB root volume. The EFS filesystem is NFS mounted to each node at `${efs_mount}`.
+  3. DNS entries in Route53 for each EC2 node.
+
+## Software Layout
+
+This Terraform configuration:
+
+  1. Downloads, if necessary, the Apache Maven `${maven_version}` binary tarball to `${efs_mount}/sources`, then untars it to `${efs_mount}/apache-maven/apache-maven-${maven_version}`
+  2. Downloads, if necessary, the Apache Zookeeper `${zookeer_version}` binary tarball to `${efs_mount}/sources`, then untars it to `${efs_mount}/zookeeper/apache-zookeeper-${zookeeper_version}-bin`
+  3. Downloads, if necessary, the Apache Hadoop `${hadoop_version}` binary tarball to `${efs_mount}/sources`, then untars it to `${efs_mount}/hadoop/hadoop-${hadoop_version}`
+  4. Downloads, if necessary, clones the Apache Accumulo Git repo from `${accumulo_repo}` into `${efs_mount}/sources/accumulo-repo`. It switches to the `${accumulo_branch_name}` branch and builds the software using Maven, then untars the binary tarball to `${efs_mount}/accumulo/accumulo-${accumulo_version}`
```

Comment:
```suggestion
  4. Clones the Apache Accumulo Git repo from `${accumulo_repo}` into `${efs_mount}/sources/accumulo-repo`. It switches to the `${accumulo_branch_name}` branch and builds the software using Maven, then untars the binary tarball to `${efs_mount}/accumulo/accumulo-${accumulo_version}`
```

Reply:
Good catch. An earlier version download the `accumulo` and `accumulo-testing` repos vs cloning them.

Reply:
Resolved in e0a9c43.